dtmi/osconfig/networking-1.json

```diff
@@ -0,0 +1,55 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:osconfig:networking;1",
+  "@type": "Interface",
+  "displayName": "Networking",
+  "description": "Provides functionality to remotely query device networking",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "NetworkConfiguration",
+      "schema": {
+        "@type": "Object",
+        "fields": [
+          {
+            "name": "InterfaceTypes",
+            "schema": "string"

```

Comment:
Which are the valid values? would make sense to include an Enum?

Reply:
The main meaning for this field is to list network interfaces by their names, completed with their types. There are a variable number of interfaces on each device, names can be anything. This same pattern is followed on all the fields in this property. Please see the spec in the e-mail discussion for more information. Example of a value for this field: “cel2=cellular;eth0=ethernet;foo=unknown;wlan1=wifi”